pdoc/doc_pyi.py

```diff
@@ -25,10 +26,17 @@ def find_stub_file(module_name: str) -> Path | None:
     """Try to find a .pyi file with type stubs for the given module name."""
     module_path = module_name.replace(".", "/")
 
-    for dir in sys.path:
+    # Find .pyi-file in a PEP 0561 compatible stub-package
+    module_part_name = module_name.split(".")
+    module_part_name[0] = f"{module_part_name[0]}-stubs"
+    module_stub_path = "/".join(module_part_name)
+
+    for path_dir in sys.path:
```

Comment:
```suggestion
    for search_dir in sys.path:
```
maybe? Strong +1 to not overriding builtins, but `path` in the variable name is typically a `Path` object. I'd be equally happy with just `d`.

Reply:
Fixed